### PR TITLE
[Changed] Add delay to tests

### PR DESCRIPTION
**Pull request by sdepold**
In order to fix the RateLimitExceeded error, this test adds an
artificial delay to the tests.

### CODE REVIEW

**Comment by sdepold** [comment]
The delay could be determined way smarter but it works and doesn't create a too huge amount of wasted time.

**Comment by sdepold** [comment]
I merged this on my own in order to proceed the development. Feel free to comment on this PR if something does not look alright

File: apiary/Makefile [hunk 1]
```diff
@@ -70,9 +70,10 @@ publish: test
...
 test: $(CDA_BLUEPRINT) $(CPA_BLUEPRINT) $(CMA_BLUEPRINT) $(IMAGES_BLUEPRINT)
-	$(DREDD) $(CPA_BLUEPRINT) $(CPA_HOST)
-	$(DREDD) $(CDA_BLUEPRINT) $(CDA_HOST)
+	$(DREDD) $(CPA_BLUEPRINT) $(CPA_HOST) --hookfiles=./test-hooks.js
+	$(DREDD) $(CDA_BLUEPRINT) $(CDA_HOST) --hookfiles=./test-hooks.js
 	$(DREDD) $(CMA_BLUEPRINT) $(CMA_HOST) \
+		--hookfiles=./test-hooks.js \
 		-h "Authorization: Bearer $(CONTENTFUL_MANAGEMENT_API_ACCESS_TOKEN)" \
 		-m GET #-m PUT -m POST #-m DELETE
 	#$(DREDD) $(IMAGES_BLUEPRINT) $(IMAGES_HOST)
```
Comment (neonichu) [inline apiary/Makefile]: Let's also add the `—hookfiles` option here, to not forget it when we can eventually activate this.

**Comment by neonichu** [comment]
:ok_hand:

File: apiary/Makefile (hunk 1 above)
Comment (sdepold) [inline apiary/Makefile]: https://github.com/contentful/slash-developers/pull/42

**Pull request merged**